app/src/main/java/de/danoeh/antennapod/fragment/CoverFragment.java

```diff
@@ -93,7 +94,16 @@ private void loadMediaInfo() {
     }
 
     private void displayMediaInfo(@NonNull Playable media) {
-        txtvPodcastTitle.setText(media.getFeedTitle());
+        if (media.getClass() == FeedMedia.class) {
```

Comment:
Sorry for the many small reviews but I did not notice this earlier. Wouldn't that mean that the new code in `ExternalMedia`  is never used?

Reply:
@ByteHamster you are right.  I just fixed.  I tested with local files, stream and downloaded episodes.